.sling-module.json

```diff
@@ -2,6 +2,6 @@
     "name": "org.apache.sling.event",
     "version": "1.0.0",
     "description": "Apache Sling Event Module",
-    "timeout": 60,
+    "buildTimeout": 60,
```

Comment:
I think you only need something like 

```javascript
{
  "jenkins": {
    "buildTimeout": 60
  }
}
```

The rest of the properties listed in there are not used